packages/core/src/LottieView/index.web.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { forwardRef, useImperativeHandle, useRef, useCallback } from 'react';
+import type { LottieViewProps } from '../types';
+import { parsePossibleSources } from './utils';
+
+let DotLottiePlayer, Player;
+try {
+    DotLottiePlayer = require('@dotlottie/react-player').DotLottiePlayer;
+    Player = require('@lottiefiles/react-lottie-player').Player;
+} catch (e) { }
```

Comment:
Isn't it better to add warning here as well to make sure they have installed the required deps?

Reply:
Ah my bad. It's being handled down here in this file.